tools/defaults.bzl

```diff
@@ -3,75 +3,11 @@
 These set common default attributes and behaviors for our local repo
 """
 
-load(
-    "@build_bazel_rules_nodejs//:index.bzl",
-    _COMMON_REPLACEMENTS = "COMMON_REPLACEMENTS",
-    _nodejs_test = "nodejs_test",
-    _pkg_npm = "pkg_npm",
-)
 load("@rules_codeowners//tools:codeowners.bzl", _codeowners = "codeowners")
 load("@rules_pkg//:pkg.bzl", _pkg_tar = "pkg_tar")
-load("//third_party/github.com/bazelbuild/bazel-skylib:rules/copy_file.bzl", "copy_file")
 
-nodejs_test = _nodejs_test
 pkg_tar = _pkg_tar
```

Comment:
guessing this file can go as well if the only thing in here is pkg_tar which we don't need anymore if we switch to release.yaml

Reply:
ahh. there is codeowners below as well. do we still want that in this repo @alexeagle ?

Reply:
`pkg_tar` is also used to create a tar of the `@bazel/runfiles`, for now I'm just changing that to a `filegroup` and we can think about it again if/when we ever publish a new `@bazel/runfiles`?